src/content/structured/components/cards/code.mdx

```diff
@@ -785,9 +782,9 @@ export const expandable = [
   {
     language: "Web component",
     snippet: `<ic-card 
-  heading="Card heading"
-  subheading="Card subheading" 
-  message="This is an example of an interactive card with an expandable content area."
+  heading="Coffee" 
+  subheading="What is coffee?" 
+  message="Coffee is a brewed drink."
```

Comment:
This message being short is causing the card to grow when you expand to see the other content. Think it's an issue with the component, but for now, if we just make this message longer than the expanded content.